portafly/package.json

```diff
@@ -5,6 +5,9 @@
   "dependencies": {
     "@patternfly/react-core": "^3.124.1",
     "@patternfly/react-table": "^2.25.6",
+    "@types/node-polyglot": "^0.4.34",
+    "moment": "^2.24.0",
```

Comment:
this is here only to test some dates, later I'll remove it

Reply:
better do the real thing with `date-fns`, that way you'll proving it works or any other findings